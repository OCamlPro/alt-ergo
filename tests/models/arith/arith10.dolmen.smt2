(set-option :produce-models true)
(set-logic ALL)
(declare-const x Real)
(declare-const y Real)
(assert (< x 2.0))
(assert (<= y 10.0))
(maximize x)
(maximize y)
(check-sat)
(get-model)
(get-objectives)
