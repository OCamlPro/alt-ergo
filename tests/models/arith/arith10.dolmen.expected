
unknown
(
  (define-fun x () Real 1.0)
  (define-fun y () Real 10.0)
)
(objectives 
  (x (- 2.0 epsilon))
  (y 10.0)
)